dnf/cli/commands/__init__.py

```diff
@@ -199,7 +199,6 @@ def set_argparser(cls, parser):
 
     def configure(self):
         demands = self.cli.demands
-        demands.fresh_metadata = False
```

Comment:
What about to set demands.fresh_metadata=True or only remove the line because default is True?

Then  dnf list/info command will work exactly as "dnf repoquery"

Reply:
It is of course possible. But it means checking / updating cache before each list/info command. It makes sense when you are interested in updates, but for the vast majority of 'dnf list/info' usage it will just slow things down.